regression/regression_var.sh

```diff
@@ -138,19 +138,17 @@ case $machine in
 
     export noscrub=/lfs1/NESDIS/nesdis-rdo2/$LOGNAME/noscrub
     export ptmp=/lfs1/NESDIS/nesdis-rdo2/$LOGNAME/ptmp
-    export fixcrtm="/lfs1/NESDIS/nesdis-rdo2/David.Huber/save/CRTM_REL-2.2.3/crtm_v2.2.3/fix_update"
-    export casesdir="/lfs1/NESDIS/nesdis-rdo2/David.Huber/save/CASES"
+#   export fixcrtm="/lfs1/NESDIS/nesdis-rdo2/David.Huber/save/CRTM_REL-2.2.3/crtm_v2.2.3/fix_update"
```

Comment:
The regression tests will need CRTM. If this is commented out, where to find the correct CRTM files?

Reply:
The CRTM module sets the variable CRTM_FIX, which is referenced in each of the tests that require it.  For instance, at line 16 of global_3dvar.sh:
`fixcrtm=${fixcrtm:-$CRTM_FIX}`
So now `fixcrtm` does not need to be set in regression_var.sh.  I followed the example under Hera in commenting the line out instead of removing it, but I think now that it makes more sense to remove it.

Reply:
Deleted the `fixcrtm` line.